docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -25,120 +34,120 @@ template parameters, Coder assigns an existing instance, reducing setup time sig
+1. The preset parameters define the base configuration and are immutable after they're claimed.
+1. Prebuilds help presets deliver even faster workspace creation.
 
-   Prebuilt workspaces can only be utilized by the default organization.
+_Note: In future releases, we will allow operators to invalidate their prebuilt workspaces programmatically._
 
-   [coder/internal#364](https://github.com/coder/internal/issues/364)
+## Administration and troubleshooting
 
-- Autoscaling
+### Managing resource quotas
 
-   Prebuilt workspaces remain running until they are claimed.
-   We do not currently have an autoscaling mechanism to reduce the number of instances after working hours.
+Prebuilt workspaces can be used in conjunction with [resource quotas](../../users/quotas.md).
+Because unclaimed prebuilt workspaces are owned by the `prebuilds` user, you can:
 
-   [coder/internal#312](https://github.com/coder/internal/issues/312)
+1. Configure quotas for any group that includes this user.
+1. Set appropriate limits to balance Prebuild availability with resource constraints.
+1. Monitor quota utilization through Coder's dashboard.
 
-## Gotchas
+If a quota is exceeded, the prebuilt workspace will fail provisioning the same way other workspaces do.
 
-### Resource Replacement
+### Template configuration best practices
 
-When a prebuilt workspace is created, it is initially [owned](#workspace-ownership) by the `prebuilds` user and a random name
-is generated for it.
-When `terraform apply` runs, it will provide these values during provisioning in the
-[`coder_workspace`](https://registry.terraform.io/providers/coder/coder/latest/docs/data-sources/workspace) and
-[`coder_workspace_owner`](https://registry.terraform.io/providers/coder/coder/latest/docs/data-sources/workspace_owner)
-datasources.
+#### Preventing resource replacement
 
-Once a prebuilt workspace is claimed, the ownership of that workspace changes to the requesting user and
-`terraform apply` is run again, now with updated values for the aforementioned datasources.
+When a prebuilt workspace is claimed, Terraform runs again with new values for the workspace owner and name.
 
-If your template has used these datasources in immutable fields (i.e. the
-[`user_data`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#user_data-1) field in
-an `aws_instance` resource), Terraform will interpret these changes as _drift_ and will therefore destroy and recreate
-the resource.
+This can cause issues:
 
-This is obviously undesirable because the prebuilt workspace will now have to provision _again_, while the user waits,
-eliminating the value of the prior pre-provisioning.
+1. The workspace is initially created with values from the `prebuilds` user and a random name.
+1. After claiming, the workspace owner and name change, which Terraform sees as configuration drift.
```

Comment:
This is too specific. The owner & name change are two of a very large set of possible changes.